content/docs/get-started/azure/create-project.md

```diff
@@ -34,9 +34,6 @@ $ mkdir quickstart && cd quickstart
 $ pulumi new azure-python
 ```
 
-> If this is your first time running `pulumi new` or most other `pulumi` commands, you will be prompted to log in to the [Pulumi service](https://app.pulumi.com). The [Pulumi CLI]({{< relref "/docs/reference/cli" >}}) works in tandem with the Pulumi service in order to deliver a reliable experience. It is free for individual use, with [additional features available for teams](https://www.pulumi.com/pricing/). Hitting `ENTER` at the prompt opens up a web browser allowing you to either sign in or sign up.
```

Comment:
I was thinking more about the change in #1740 and was wondering if we should keep the note here as well. Since someone might not have used the CLI yet in the configure step (if they've set their cloud authentication in a way that doesn't use the Pulumi CLI).

So how about we add `{{< cli-note >}}` here an all the `create-project` steps (including AWS).